neo4j-app/neo4j_app/app/graphs.py

```diff
@@ -24,30 +25,50 @@ def graphs_router() -> APIRouter:
     async def _graph_dump(
         project: str,
         payload: DumpRequest,
+        request: Request,
     ) -> StreamingResponse:
-        with log_elapsed_time_cm(
```

Comment:
Log was remove on purpose here, the elapsed time here can't be correclty computed as we're streaming, this returns right away